traffic_ops/testing/api/v1/parameters_test.go

```diff
@@ -45,6 +45,20 @@ func CreateTestParameters(t *testing.T) {
 
 }
 
+func CreateMultipleTestParameters(t *testing.T) {
+
+	pls := []tc.Parameter{}
+	for _, pl := range testData.Parameters {
+		pls = append(pls, pl)
+	}
+	resp, _, err := TOSession.CreateMultipleParameters(pls)
+	log.Debugln("Response: ", resp)
```

Comment:
When trying to run the API tests, I get this error:

```dockerfile
Step 12/25 : RUN go test -c ./v1* -o traffic_ops_integration_test
 ---> Running in 32f467ccc60b
# github.com/apache/trafficcontrol/traffic_ops/testing/api/v1 [github.com/apache/trafficcontrol/traffic_ops/testing/api/v1.test]
v1/parameters_test.go:55:2: undefined: log
v1/profile_parameters_test.go:111:2: undefined: log
ERROR: Service 'integration' failed to build: The command '/bin/sh -c go test -c ./v1* -o traffic_ops_integration_test' returned a non-zero code: 2
```

Reply:
done